message_center_compassion/models/gmc_message.py

```diff
@@ -256,14 +256,21 @@ def _perform_incoming_action(self):
         and call the process_commkit method on the related object. """
         object_ids = list()
         action = self.mapped('action_id')
+        success = False
```

Comment:
Not sure that all these changes are needed in this file. Doesn't it work without it ? `object_ids` won't evaluate to false if it is `[-1]`

Reply:
No it doesn't work without the code, there is an error when computing object_ids on line 268 because it tries to find a hold with id == -1

Reply:
But maybe success is a bad choice of name for the variable